apps/web/src/pages/ApplicantDashboardPage/components/QualifiedRecruitments/QualifiedRecruitmentCard.tsx

```diff
@@ -0,0 +1,417 @@
+        <p>
+          {intl.formatMessage({
+            defaultMessage:
+              "You have chosen not to appear in talent results for this recruitment.",
+            id: "bqWmTD",
+            description:
+              "Suspended recruitment status message on qualified recruitment card.",
+          })}
+          {/* Dialog that allows user to un-suspend the qualified recruitment (application). Sets suspendedAt date to null. */}
+          <QualifiedRecruitmentDialog
+            id={id}
+            isSuspended={!!suspendedAt}
+            closeButtonLabel={intl.formatMessage({
+              defaultMessage: "Yes, add me to search results",
+              id: "V8GyRL",
+              description:
+                "Dialog close button on change search results status from qualified recruitment section.",
+            })}
+            openDialogLabel={intl.formatMessage({
+              defaultMessage: "I want to appear in results again",
+              id: "XHfbPp",
+              description:
+                "Open dialog text on change search results status from qualified recruitment section.",
+            })}
+            primaryBodyText={intl.formatMessage({
+              defaultMessage:
+                "You are currently not showing up in talent search results for the “IT-01 Helpdesk support technician” pool.",
+              id: "veIkpN",
+              description:
+                "Dialog main body on change search results status from qualified recruitment section.",
+            })}
+            secondaryBodyText={intl.formatMessage({
+              defaultMessage:
+                "By re-adding yourself to this recruitment processes results, managers will once again be able to request your profile as a part of talent requests. You can always remove yourself again at a later time.",
+              id: "EGwR9h",
+              description:
+                "Dialog main body on change search results status from qualified recruitment section.",
+            })}
+            title={intl.formatMessage({
+              defaultMessage: "Change your appearance in search results",
+              id: "FGF3sd",
+              description:
+                "Dialog title on change search results status from qualified recruitment section.",
+            })}
+          />
+          .
+        </p>
+      </div>
+    );
+  };
+
+  // Essential Skills
+  const skills = application.poolAdvertisement?.essentialSkills
+    ? application.poolAdvertisement?.essentialSkills
+    : [];
+
+  // Conditionals for qualified recruitment card actions
+  const applicationIsDraft = isDraft(application.status);
+  const recruitmentIsExpired = isExpired(
+    application.status,
+    application.poolAdvertisement?.closingDate,
+  );
+  const isApplicantPlaced = isPlaced(application.status);
+  return (
+    <div
+      data-h2-background="base(foreground)"
+      data-h2-shadow="base(medium)"
+      data-h2-radius="base(0px rounded rounded 0px)"
```

Comment:
This is an issue with `ApplicationCard` as well. Looks like these cards should be rounded on every corner.

```suggestion
      data-h2-radius="base(rounded)"
```

Reply:
Nice! This commit [8ba4ced](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/8ba4ced914e4606afece9a57b25840d541985511) fixes both card component.